### PR TITLE
Doc fix: Editing README.md

### PR DESCRIPTION
**Pull request by ogorbacheva**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category:
- Doc fix.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,56 +1,56 @@
-# How to contribute to ClickHouse documentation?
+# How to Contribute to ClickHouse Documentation
 
-Basically ClickHouse uses "documentation as code" approach, so you can edit Markdown files in this folder from GitHub web interface or fork ClickHouse repository, edit, commit, push and open pull request.
+ClickHouse uses the "documentation as code" approach, so you can edit Markdown files in this folder from the GitHub web interface. Alternatively, fork the ClickHouse repository, edit, commit, push, and open a pull request.
 
-At the moment documentation is bilingual in English and Russian, so it's better to try keeping languages in sync if you can, but it's not strictly required as there are people watching over this. If you add new article, you should also add it to `toc_{en,ru,zh,fa}.yaml` files with pages index.
+At the moment documentation is bilingual in English and Russian. It's best to try to keep both languages in sync if you can, but this is not strictly required because there are people watching over this. If you add a new article, you should also add it to `toc_{en,ru,zh,fa}.yaml` files with the pages index.
```
Comment (blinkov) [inline docs/README.md]: * both -> all
* "this ... this" sounds not so good

File: docs/README.md [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
 
-## Target audience
+## Target Audience
 
-When you write pretty much any text, first thing you should think about: who exactly will read it and in which terms it is better to "talk" with them.
+When you write pretty much any text, the first thing you should think about is who will read it and which terms you should use for communicating with them.
 
-ClickHouse can be directly used by all sorts of either analysts and engineers, so you should only basic technical background of reader when writing content for generic parts of documentation, like query language, tutorials or overviews. Though it is ok for articles describing ClickHouse internals, guides for operating ClickHouse clusters, contributing to C++ code and other similar topics.
+ClickHouse can be directly used by all sorts of analysts and engineers. For generic parts of documentation (like the query language, tutorials or overviews), assume that the reader only has a basic technical background. For more technical sections (like articles that describe ClickHouse internals, guides for operating ClickHouse clusters, or rules for contributing to C++ code), you can use specific technical language and concepts.
```
Comment (blinkov) [inline docs/README.md]: I'd replace "specific" with some synonym(s) because next section is called "Specific ..."

File: docs/README.md [hunk 3]
```diff
@@ -1,56 +1,56 @@
...
-* Hide block to be opened by click: `<details markdown="1"> <summary>Header</summary> hidden content</details>`.
+* Lists are on lines starting with `* unordered` or `1. ordered`, but there should be an empty line before the first list item. Sub-lists must be indented with 4 spaces.
+* Inline code fragments are <code>&#96;in backticks&#96;</code>.
+* Multiline code blocks are <code>&#96;&#96;&#96;in triple backtick quotes &#96;&#96;&#96;</code>.
+* Brightly highlighted text starts with  `!!! info "Header"`, followed by 4 spaces on the next line and content. For a warning, replace `info` with `warning`.
+* Hidden block that opens on click: `<details markdown="1"> <summary>Header</summary> hidden content</details>`.
 * Colored text: `<span style="color: red;">text</span>`.
-* Additional anchor to be linked to: `<a name="my_anchor"></a>`, for headers fully in English they are created automatically like `"FoO Bar" -> "foo-bar"`.
+* Heading anchor to be linked to: `Title {#anchor-name}`. Anchors are created automatically for headings written fully in English: `"Title Name" -> "title-name"`.
```
Comment (blinkov) [inline docs/README.md]: "written fully in English" part is not relevant anymore

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ Some additional configuration has to be done to actually make a new language liv
 * Brightly highlighted text starts with  `!!! info "Header"`, followed by 4 spaces on the next line and content. For a warning, replace `info` with `warning`.
 * Hidden block that opens on click: `<details markdown="1"> <summary>Header</summary> hidden content</details>`.
 * Colored text: `<span style="color: red;">text</span>`.
-* Heading anchor to be linked to: `Title {#anchor-name}`. Anchors are created automatically for headings written fully in English: `"Title Name" -> "title-name"`.
```
Comment (ogorbacheva) [inline docs/README.md]: I offer to hide info about anchor generation. It's best to specify anchors explicitly.

**Pull request merged**